noah_frontend/src/components/donate.js

```diff
@@ -84,7 +89,8 @@ class DonateView extends React.Component {
         let errors = {}
         const phoneRegex = /^\d{10}$/,
               pinCodeRegex = /^\d{6}$/,
-              landPhoneRegex = /^0\d{10}$/
+              landPhoneRegex = /^0\d{10}$/,
```

Comment:
` /^0?\d{10}$/` should be enough to cover both Indian landline and mobile numbers

Reply:
Make sense. Will update it